strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/nuget/NugetArtifactControllerTest.java

```diff
@@ -65,6 +74,8 @@
     @Inject
     private ArtifactEntryService artifactEntryService;
 
+    private MockMvc mvc;
```

Comment:
@anki2189 can you please use restassured `mockMvc` instead, like it was before and in all other tests?

Reply:
restassured `mockMvc` does not support multipart file with `PUT` method.

Reply:
this is interesting, let me check

Reply:
yeah, seems your'e right
this probably was the reason of `NugetArtifactControllerTest.createPushRequest` method was used thre, it doesn't  works for you this way?

Reply:
No, it was not working .

Reply:
we should make it work this way. for what reason does this not work?

Reply:
Getting {"error":"Required request part 'package' is not present"}

Reply:
Can you please leave `NugetArtifactControllerTest.createPushRequest` method and make failed tests ignored? So we we will create follow up task to fix it.

Reply:
BTW, why we are hesitating to use springframework `MockMvc`?

Reply:
because currently we use REST Assured, it will be not good to mix it with  Spring MockMvc.

Reply:
Changes done as suggested.

Reply:
great, thanks!
@steve-todorov can you please check the `s-w-i-t` with `Mono JIT compiler version 6.4.0.198`?

Reply:
@ anki2189 Can you please remove Spring Mock MVC, as we agreed earlier?

Reply:
alsi, disabled `testPackageCommonFlow` test should be the only change here

Reply:
`testPackageCommonFlow` , `testChocolatey`, `testLastVersionPackageSearch` are 3 test cases needs to be disabled as these are using `createPushRequest`